.eslintrc.json

```diff
@@ -52,22 +52,40 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": [
-          "**/stories/**/*.*",
-          "**/.storybook/**/*.*"
-        ],
+        "devDependencies": ["**/stories/**/*.*", "**/.storybook/**/*.*"],
         "peerDependencies": true
       }
     ]
   },
   "overrides": [
     {
-      "files": "tests/**/*.ts",
+      "files": ["**/*.test.{ts,tsx,js,jsx}", "**/__tests__/**/*.{ts,tsx,js,jsx}"],
       "env": {
         "node": true,
-        "jest": true
+        "jest": true,
+        "jest/globals": true
+      },
+      "plugins": ["jest", "no-snapshot-testing", "testing-library"],
+      "extends": [
```

Comment:
I think we only need 
```
        "plugin:jest/recommended",
        "plugin:testing-library/react"
```
here as the rest of them already come from the main config